c2corg_ui/static/js/user.js

```diff
@@ -100,4 +100,12 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
+ * @export
+ */
+app.UserController.prototype.hasEditRightsOuting = function(users) {
+  this.auth.hasEditRightsOuting(users);
```

Comment:
@asaunier, it is not possible to call services directly from directive templates.
So this looks consistent with the angularjs philosophy.
Do you know of another way?